modules/gitops-release-notes-1-12-2.adoc

```diff
@@ -0,0 +1,68 @@
+
+** Logs for Argo Rollouts Operator pods and Argo Rollouts pods.
+** Contents of Argo Rollouts Manager and Argo Rollouts custom resources (CRs).
+** Contents of `Deployment`, `Statefulset`, `ConfigMaps` CRs created by the `must-gather` tool in the Argo Rollouts Operator.
+link:https://issues.redhat.com/browse/GITOPS-3947[GITOPS-3947]
+
+[id="fixed-issues-1-12-2_{context}"]
+== Fixed issues
+
+* Before this update, users could not use the `argocd-k8s-auth` binary to add GKE and EKS clusters because this binary was not available in the {gitops-shortname} container. This update fixes the issue by adding the `argocd-k8s-auth` binary in the {gitops-shortname} container. link:https://issues.redhat.com/browse/GITOPS-4226[GITOPS-4226]
+
+* Before this update, attempts to connect to Azure DevOps with Argo CD would result in an error due to the deprecation of the `rsa-ssh` host key algorithm by the Azure DevOps Repository service. This update fixes the issue by providing support for the `rsa-ssh` host key algorithms during the communication process between Argo CD and Azure DevOps Repository service. link:https://issues.redhat.com/browse/GITOPS-4543[GITOPS-4543]
+
+* Before this update, {gitops-shortname} console plugin workloads did not schedule on infrastructure nodes when the `runOnInfra` field was enabled in the `GitOpsService` custom resource (CR). This update fixes the issue by adding the infrastructure `node-selector` on the {gitops-shortname} console plugin workloads. This enables users to configure custom node-selectors and tolerations on the `gitops-console` pod. As a result, when the `runOnInfra` field is enabled, the {gitops-shortname} console plugin pod is placed on the infrastructure nodes like the other default workloads. link:https://issues.redhat.com/browse/GITOPS-4496[GITOPS-4496]
+
+* Before this update, the `ignoreDifferences` sync option in Argo CD did not work for array fields. This update fixes the issue by modifying the merge strategy of the `ignoreDifferences` sync option used in the upstream project to handle array fields. As a result, the sync option now functions correctly by allowing users to ignore specific elements in the array during sync. link:https://issues.redhat.com/browse/GITOPS-2962[GITOPS-2962]
+
+* Before this update, users were unable to include context for notifications in the `NotificationsConfiguration` custom resource (CR). With this update, users can now add context for notifications by using the `context` field in the `NotificationsConfiguration` CR. This field can also be used to establish shared context across all templates defined in the `NotificationsConfiguration` CR. link:https://issues.redhat.com/browse/GITOPS-4303[GITOPS-4303]
++
+Example of the `context` field in the `NotificationsConfiguration` CR
++
+[source,yaml]
+----
+spec:
+  context:
+    region: east #<1>
+----
+<1> Context between all notification templates is in `key-value` pairs
```

Comment:
```suggestion
<1> Context among all notification templates is in `key-value` pairs
```
@Dhruv-Soni11 Should we not replace between with among here? WDYT?
Please refer to the usage of between and among, talk to the SME and implement what wording is suitable here.

Reply:
Keeping it as among in this instance.